packages/lambda-tiler/src/util/response.ts

```diff
@@ -1,5 +1,7 @@
-import { LambdaHttpResponse } from '@linzjs/lambda';
+import { LambdaHttpRequest, LambdaHttpResponse } from '@linzjs/lambda';
 
 export const NotFound = (): LambdaHttpResponse => new LambdaHttpResponse(404, 'Not Found');
 export const NotModified = (): LambdaHttpResponse => new LambdaHttpResponse(304, 'Not modified');
 export const NoContent = (): LambdaHttpResponse => new LambdaHttpResponse(204, 'No Content');
+export const OkResponse = (req: LambdaHttpRequest): LambdaHttpResponse =>
+  new LambdaHttpResponse(200, 'ok').json({ id: req.id, correladtionId: req.correlationId, message: 'ok' });
```

Comment:
I think you misspelled  correlation

Reply:
:1st_place_medal: Good pick.

Reply:
I use a code spell checker so it highlights spelling issues like this 
![image](https://user-images.githubusercontent.com/1082761/182971910-580bf165-0b3c-4732-9d96-4a56e4ebf2d8.png)

Reply:
Wait, did you do reviews in vs code?

Reply:
ahh Nope, but it would highlight this issue to you before you make a PR :grin:

Reply:
I got the spell checker as well just not working sometimes?
![image](https://user-images.githubusercontent.com/12163920/182972257-254354f4-ffbd-4a48-b98a-6f609b323131.png)